akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerStressSpec.scala

```diff
@@ -21,7 +21,7 @@ object CircuitBreakerStressSpec {
   case class Result(doneCount: Int, timeoutCount: Int, failCount: Int, circCount: Int)
 
   class StressActor(breaker: CircuitBreaker) extends Actor with ActorLogging with PipeToSupport {
-    import context.dispatcher
+    private implicit val ec: ExecutionContext = context.dispatcher
```

Comment:
This is used in `pipeTo` which happens in receive, and `scheduleOnce` which also happens on receive, so here `import context.dispatcher` is safe.